cvise/utils/testing.py

```diff
@@ -479,14 +468,14 @@ def run_parallel_tests(self):
                     self.terminate_all(pool)
                     return success
 
-                folder = tempfile.mkdtemp(prefix=self.TEMP_PREFIX, dir=self.root)
+                folder = Path(tempfile.mkdtemp(prefix=self.TEMP_PREFIX, dir=self.root))
                 test_env = TestEnvironment(
                     self.state,
                     order,
                     self.test_script,
                     folder,
                     self.current_test_case,
-                    self.test_cases ^ {self.current_test_case},
+                    self.test_cases,
```

Comment:
Same thing here.  This looks intentionally done, so maybe it doesn't matter to CVise that the first one is repeated.